src/plugins/minimal_scene/EngineToQtInterface.hh

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef IGNITION_GUI_PLUGINS_ENGINETOQTINTERFACE_HH_
+#define IGNITION_GUI_PLUGINS_ENGINETOQTINTERFACE_HH_
+
+#include <ignition/rendering/RenderTypes.hh>
+
+#include "ignition/gui/qt.h"
+
+#include <memory>
+
+namespace ignition
+{
+namespace gui
+{
+namespace plugins
+{
+  class EngineToQtInterfacePrivate;
+
+  /// \brief
+  class EngineToQtInterface
```

Comment:
missing docs explaining class and methods below

Reply:
Oh you're right! Updated the PR vastly expanding the documentation. I apologize for being so sloppy with this PR's documentation!